### PR TITLE
Expand supported profiler docs with rendering of output files

### PR DESCRIPTION
**Pull request by jfy133**
* [ ] fix #(issue number)
* [x] description of feature/fix
* [ ] tests added/passed
* [ ] add an entry to the [next release](../CHANGELOG.rst)

In addition to the table, given some outputs don't actually have the headernames - therefore it might be useful for the user to have a visual example of what the output looks like

### CODE REVIEW

**Comment by jfy133** [comment]
Supersedes: https://github.com/taxprofiler/taxpasta/pull/52

File: docs/supported_profilers/kaiju.md [hunk 1]
```diff
@@ -4,12 +4,22 @@
...
+| ------------- | ----------- |
 | file          |             |
 | percent       |             |
 | reads         |             |
 | taxon_id      |             |
 | taxon_name    |             |
+
+## Example
+
+```bash
```
Comment (Midnighter) [inline docs/supported_profilers/kaiju.md:19]:
```suggestion
```text
```

File: docs/supported_profilers/diamond.md [hunk 2]
```diff
@@ -7,7 +7,39 @@
...
+escherichia_coli_308/1	511145	1.55e-12
+shigella_dysenteriae_1418/1	1310613	7.75e-10
+escherichia_coli_962/1	1310613	7.02e-06
+shigella_dysenteriae_520/1	1310613	5.46e-12
+shigella_dysenteriae_242/1	1310613	2.91e-12
+escherichia_coli_1146/1	1310613	2.61e-11
+escherichia_coli_551/1	1310613	1.94e-14
+shigella_dysenteriae_1094/1	1310613	1.74e-13
+shigella_dysenteriae_999/1	0	0
+```
```
Comment (Midnighter) [inline docs/supported_profilers/diamond.md:45]: Looks duplicated?

```suggestion
```

**Review by Midnighter — COMMENTED** [review]
Thanks for the examples. Two small comments.

**Comment by jfy133** [comment]
I'll let you merge, it says rebase again and I'm scared

**Pull request merged**